docs/robotframework.rst

```diff
@@ -200,6 +200,180 @@ In addition to browser interactions, the Salesforce Library also provides the fo
+import the **PageObjects** library and any custom page object files you
+wish to use.
+
+The second thing you must do is either explicitly load the keywords
+for a page object, or reference a page object with one of the generic
+keywords provided by the **PageObjects** library.
+
+To explicitly load the keywords for a page object you can use the
+:code:`load page object` keyword provided by the **PageObjects**
+library. Other keywords provided by that library will automatically
+import the keywords if they are successful. For example, you can call
+:code:`Go To Page` followed by a page object reference, and if that page is
+able to be navigated to, its keywords will automatically be loaded.
+
+Page Object Keywords
+--------------------
+
+The **PageObjects** library provides the following keywords:
+
+* Current Page Should Be
+* Go To Page Object
+* Load Page Object
+* Log Page Object Keywords
+
+Current Page Should Be
+^^^^^^^^^^^^^^^^^^^^^^
+
+Example: :code:`Current Page Should Be  Listing  Contact`
+
+This keyword will attempt to validate that the given page object
+represents the current page. Each page object may use its own method
+for making the determination, but the built-in page objects all
+compare the page location to an expected pattern
+(eg: _.../lightning/o/..._). If the assertion passes, the keywords for
+that page object will autoamtically be loaded.
+
+This keyword is useful if you get to a page via a button or some other
+form of navigation, in that it allows you to both assert that you are
+on the page you think you should be on, and load the keywords for that
+page, all with a single statement.
+
+Go To Page Object
+^^^^^^^^^^^^^^^^^
+
+Example: :code:`Go to page object  Listing  Contact`
+
+This will attempt to go to the listing page for the Contact object,
+and then load the keywords for that page.
+
+Log Page Object Keywords
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example: :code:`Load Page Object Keywords`
+
+This keyword is primarily a debugging tool. When called it will log
+each of the keywords for the current page object.
+
+Load Page Object
+^^^^^^^^^^^^^^^^
+
+Example: :code:`Load page object  Listing  Contact`
+
+This will load the page object for the given **page_type** and
+**object_name_**. It is useful when you want to use the keywords from a
+page object without first navigating to that page (ie: when you are
+already on the page and don't want to navigate away).
+
+
+Generic Page Objects
+--------------------
+
+You do not need to create a page object in order to take advantage of
+the new page object keywords. If you use one of the page object
+keywords for a page that does not have its own page object, the
+**PageObjects** library will try to find a generic page.
+
+For example, if you use :code:`Current page should be  Home  Event` and
+there is no page object by that name, a generic :code:`Home` page object
+will be loaded, and it's object name will be set to :code:`Event`.
```

Comment:
```suggestion
will be loaded, and its object name will be set to :code:`Event`.
```

Reply:
ugh. I hate it when I make this mistake.